media-source-respec.html

```diff
@@ -705,7 +705,8 @@
           <h4>Seeking</h4>
           <p>Run the following steps as part of the "<i>Wait until the user agent has established whether or not the media data for the new playback position is available, and, if it is, until it has decoded enough data to play back that position"</i> step of the <a def-id="hme-seek-algorithm"></a>:</p>
           <ol>
-            <li>The media element looks for <a def-id="media-segments"></a> containing the <var>new playback position</var> in each <a>SourceBuffer</a> object in <a def-id="activeSourceBuffers"></a>.
+            <li>The media element looks for <a def-id="media-segments"></a> containing the <var>new playback position</var> in each <a>SourceBuffer</a> object in <a def-id="activeSourceBuffers"></a>. Any position within a <a def-id="timerange"></a> in the current value of the <a def-id="hme-buffered"></a> attribute MUST have a buffered media segment for that position.
```

Comment:
@jdsmith3000 I wasn't proposing a new step, rather clarifying the existing one. Are you proposing a conditional sub-step like:
Existing step text + "If readyState==ended, then:
1)"[ something like your text ]

I agree the text in the note that follows should be normative, so putting in outside of the note would be better. I'll proceed with a conditional sub-step update to this PR.